src/useComputed.ts

```diff
@@ -1,7 +1,14 @@
 import * as mobx from "mobx"
 import { useMemo } from "react"
 
+let warned = false
+
 export function useComputed<T>(func: () => T, inputs: ReadonlyArray<any> = []): T {
+    if (process.env.NODE_ENV !== "production" && !warned) {
+        warned = true
+        // tslint:disable-next-line: no-console
+        console.warn("[mobx-react-lite] useComputed has been deprecated")

```

Comment:
```suggestion
        console.warn("[mobx-react-lite] useComputed has been deprecated. Use useLocalStore instead.")
```

Reply:
So you don't agree with this suggestion? I mean there should be some recommendation when deprecating something.